README.md

```diff
@@ -1,6 +1,14 @@
 ## Credits
 [This repository it a fork from original tymonsdesigns/jwt-auth](https://github.com/tymondesigns/jwt-auth/wiki), we decided to fork and work independent because the original one was not being updated for long time and keep doing support for the application.
 
+## Migrating from [`tymondesigns/jwt-auth`](https://github.com/tymondesigns/jwt-auth)
+
+This uses different namespace, then `tymondesigns/jwt-auth`, but overall, provides the same API, that makes migration to this repository pretty easy:
+
+1) Replace `"tymon/jwt-auth": "^1.0"` with `"php-open-source-saver/jwt-auth": "^1.2"` in your `composer.json` 
+2) Run `composer update`
```

Comment:
> One downside of running `composer update` this way is that it will upgrade _all_ packages

I did some tests to figure out what would have the least amount of impact on existing packages:
1. `composer update` works, but updates all packages which may not be desired
2. Running first `composer remove tymon/jwt-auth` and then `composer require php-open-source-saver/jwt-auth` seems to work nicely, one does not even have to manually touch the `composer.json`
3. Keeping manually changing the `composer.json` but still avoiding updating unrelated packages, this also worked in my test: `composer update php-open-source-saver/jwt-auth --with-dependencies`
  Since `tymon/jwt-auth` is not referenced in `composer.json`, this **will remove** it too

My preference would be 2): even though it's not as nice looking and is essentially two commands, it does not require manually changing anything and will automatically chose the best version to install (`^1.2` currently) so is "maintenance free" insofar we don't need to update the reference in the README regarding the version number:

```suggestion
1) Run `composer remove tymon/jwt-auth`
2) Run `composer require php-open-source-saver/jwt-auth`
```

WDYT?

Reply:
I'll re-check once again, might be possile, that it is needed to add `--no-scripts`, since if composer will trigger package discovery, it can throw an error about non-existing classes

Reply:
In my case, special snowflake scenario, I've actually disabled the auto-discovery for the (previous name of the) package because I register my own serviceprovider, which extends the one from the package.